django_mongodb/fields/object_id.py

```diff
@@ -0,0 +1,37 @@
+from bson import ObjectId
+from bson.errors import InvalidId
+from django.core import exceptions
+from django.db.models.fields import Field
+from django.utils.translation import gettext_lazy as _
+
+
+class ObjectIdField(Field):
+    default_error_messages = {
+        "invalid": _("“%(value)s” value must be an Object Id."),
+    }
+    description = _("ObjectId")
+
+    def get_internal_type(self):
```

Comment:
The common bits of this class should be in a separate ObjectIdMixin so ObjectIdAutoField can share it.